### PR TITLE
Blog - CIOs (and CDOs) with CFOs to make digital transformation a reality

### PR DESCRIPTION
**Pull request by remyb2canada**
Blog attempting to narrate how CIOs are heavily made accountable for delivery (as per the new Policy on Service and Digital) but that they need CFOs to be successful (as project management and risk management are domains of the CFO).

For those not git/jekyll-initiated (*cough* *cough* Phil): https://014gc-my.sharepoint.com/:w:/g/personal/remy_bernard_hrsdc-rhdcc_gc_ca/ESluoH7mcz1IqjXxpnde3sEB_2WIPEkfPPLX9dexEJRe1g?e=kGBabe

### CODE REVIEW

**Review by RachelMuston — COMMENTED** [review]
Hi @remyb2canada! I'm really glad we are getting these thoughts out on the blog :-) I struggled a bit with the flow of the narrative though.  

For example...in the excerpt and the title we talk about the role of the CFO in making digital transformation a reality. And yet we don't mention the CFO anywhere in the blog post.  I was expecting to come to a section starting with..."..and this is where the CFO comes in". or something like that.  

Also the blog is covering a lot of ground. Wondering if it would be good to split into multiple blog posts?  E.g. one on the increased role of the CIO thanks to the policy on service and digital, one on the role of the CFO re: planning and investments/project management etc. and a third one tying it all together with the opportunities for change? 

Last thing...there are a lot of images and they are really complex (adds cognitive load to reading the post and because they are so complex it hard to provide a really descriptive alt text without writing pages of text). My recommendation would be to take them out or simplify them. Personally I skipped over them as they didn't add more for me than what was already in the text of the post. If the decision is to keep them in, just wanted to check....did we create them?  just thinking about copyright....

**Review by ElminaIusifova — COMMENTED** [review]
Great post @remyb2canada ! As for a debutant and not very wise person in this field, I got a little lost as it was capturing many different pieces that are difficult to digest at the same time (maybe it is just me though). I find the blog a bit lengthy.  Great images ;) where did you get them? ;) Also, your blog inspired me to start thinking about writing a blog (for the first time in my life) on how user engagement is important in building any application and service delivery.  Thanks

**Comment by remyb2canada** [comment]
> 
> 
> Great post @remyb2canada ! As for a debutant and not very wise person in this field, I got a little lost as it was capturing many different pieces that are difficult to digest at the same time (maybe it is just me though). I find the blog a bit lengthy. Great images ;) where did you get them? ;) Also, your blog inspired me to start thinking about writing a blog (for the first time in my life) on how user engagement is important in building any application and service delivery. Thanks

thanks @ElminaIusifova  :) Keeping a blog short while having enough information is really hard. I'm curious how to do this for this particular one. I agree if it's too long, people won't read it.

The images are the extent of my artistic-powerpoint talent :)

Fully support your blog ambition. GO FOR IT!

**Comment by remyb2canada** [comment]
@RachelMuston excellent feedback. I'll clarify the fit of the CFO and simplify the images (made with powerpoint :P). Agree its a bit lengthy but I would hate to remove too much content and leave it too broad. The only section I could think of removing is the part about the adverse effects of the current model (the 5 points after the image of the waterfall process). Thoughts?

**Comment by remyb2canada** [comment]
@RachelMuston pushed some changes for wordings and simplification of images.

**Comment by gcharest** [comment]
Could we do a series? Might keep the read shorter and help better structure it?

Could be a short publishing series, one day apart or the whole lot of them so that Netflix aficionados can binge them :yum:

**Comment by gcharest** [comment]
Hmm, never mind the series, it looks worse than it is in GitHub. Looks like a decent size in the rendered version IMHO

**Comment by remyb2canada** [comment]
@gcharest @RachelMuston you may be on to something.
Perhaps we start with this blog (mean to tease and attract attention, especially for CFO people), and we keep adding a few blogs throughout the next months on progress. E.g. what needs to change in Project Management? What do we mean by Product Management? How is Architecture involved in decision making? Etc.

Each of those subsequent blogs with an objective not about how we solve it, but to entice discussions on the subject

**Comment by gcharest** [comment]
That would totally make sense. But to help frame those discussions, we probably would need to end with further questions or challenges to explore?

Part of the intertwined issues will be around the procurement, architecture, data, policies, training, litigation fears, etc.

**Comment by RachelMuston** [comment]
Woah you made those images @remyb2canada?! I'm impressed! I assumed they came from a technical article or something! love the simplification of the images. Cognitive load is much reduced.

**Comment by RachelMuston** [comment]
Related to the 'series' line of thinking....this question on the free agent slack generated 34 replies 

"I've got a two part question and am looking for practitioners and experts for advice.
1. Does anyone have insights into the difference in project management, program management and product management in the context of GC digital transformation?
2. For the policy wonks, anyone familiar with policy on the planning and management of investments and how it affects an IT branch?"

I can share the replies via email (most were "...following! i'm also working in this space and trying to figure it out"). 

All that to say, folks across the GC are interested in understanding this stuff better.

File: _posts/en/2020-10-27-cio-digital-transformation.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  <tr>
+    <td>Cyber Security</td>
+    <td>6&#37;</td>
+    <td>74</td>
+  </tr>
+</table>
+
+With such a focus on CIOs and CDOs, TB formally recognizes the **ubiquitous nature of technology in delivering services** to Canadians.
+Because CIOs and CDOs are accountable for such a large number of requirements, we claim they also need to be in a position to determine _how_ technology investments are to be managed (a domain historically attributed to CFOs).
+**Elected officials** now recognize that technology is at the core of service delivery.
```
Comment (gcharest) [inline _posts/en/2020-10-27-cio-digital-transformation.md:69]: Do we infer that by the nature of the policy changes or by the mandate letter contents? (Just wondering if there's a quote we can make somewhere)

File: _posts/en/2020-10-27-cio-digital-transformation.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+This means that in order for policy makers to affect Canadians, they will need to go through technology delivery.
+If technology is not responsive enough, the lag between policy making and service delivery will directly impact the ability of departments to fulfill their mandates (which did happen in the past and made [front-page news](https://www.cbc.ca/news/politics/federal-it-systems-critical-failure-1.5448871)).
+Policy makers looking for data to inform their evidence-based decisions (i.e., user feedback loops) will be stored in databases, managed by software.
+
+At ESDC, we find the above explanation articulated as the [Business Delivery Modernization's (BDM) 2<sup>nd</sup> main objective: Policy Agility](http://blogs-blogues.prv/ntn-bnt/bdm-not-just-another-acronym-its-an-initiative-transforming-service-delivery-as-we-know-it/).
```
Comment (gcharest) [inline _posts/en/2020-10-27-cio-digital-transformation.md:74]: Any chance there's a public version of that blog?

File: _posts/en/2020-10-27-cio-digital-transformation.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+3. ***It puts IT in a passive state***: "_I need requirements_" is something you probably heard your IT teams say to you.
+This should not be surprising as we are framing the organization to behave as such.
+The [Government's pandemic response](https://nationalpost.com/opinion/john-ivison-amid-staggering-unemployment-rate-public-servants-processing-ei-claims-are-the-unsung-heroes) is an exception to this statement where personnel worked creatively together in a time of crisis.
+
+4. ***Progression is measured with documents, instead of working software***: If after 18 months of work and $2M spent we do not have working software to show for, would we consider it a good investment?
+
+> Only **successful projects, in production**, enable the organization to gain **empirical evidence** necessary for their **evidence-based decisions**.
+
+## Opportunities for change
```
Comment (gcharest) [inline _posts/en/2020-10-27-cio-digital-transformation.md:137]: Should we acknowledge challenges to the proposed approach?

Taking this road needs senior leaders to acknowledge the benefits and take key actions to tackle challenges:
Change of governance, HR strategy to acquire knowledge while maintaining existing procedures, new type of architecture which challenge the current principles in place when we acquire technology, current data management practice, etc.

**Review by gcharest — APPROVED** [review]
Just comments, I feel it's mature enough to go to translation and most comments could be tackled in further posts!

Great writing!!

**Review by gabrielcossette — APPROVED** [review]
Excellent blog Rémy! Nothing to add. I made some minor adjustments, feel free to edit.

**Pull request merged**